plugins/ZynAddSubFx/ZynAddSubFx.h

```diff
@@ -51,30 +53,39 @@ class LedCheckBox;
 class ZynAddSubFxView;
 }
 
-class ZynAddSubFxRemotePlugin : public RemotePlugin
+class ZynAddSubFxRemotePlugin
+	: public RemotePlugin
+	, public AudioPluginBufferInterface<AudioDataLayout::Split, float, 0, 2>
 {
 	Q_OBJECT
 public:
-	ZynAddSubFxRemotePlugin();
+	ZynAddSubFxRemotePlugin(PluginPinConnector* pinConnector);
 
 	bool processMessage( const message & _m ) override;
 
+	auto inputBuffer() -> SplitAudioData<float, 0> override;
+	auto outputBuffer() -> SplitAudioData<float, 2> override;
+	void updateBuffers(int channelsIn, int channelsOut) override;
 
 signals:
 	void clickedCloseButton();
 
-} ;
-
+private:
+	std::array<float*, 2> m_accessBuffer;
+};
 
 
-class ZynAddSubFxInstrument : public Instrument
+// TODO: Is it always 0 inputs, 2 outputs?
```

Comment:
I wasn't sure whether Zyn might have sidechain inputs or extra outputs that we just aren't using

Reply:
I just asked fundamental, he has been zyn maintainer of zyn since 2009. He says he remembers none, except for the multiple Part outputs, which do not make much sense inside a DAW (they are rather used in MIDI settings). So I conclude the code is fine already.